packages/internal-test-env/package.json

```diff
@@ -2,15 +2,32 @@
   "name": "@inrupt/internal-test-env",
   "version": "1.4.0",
   "description": "This package provides various test environment utilities needed for jest when using the Inrupt SDKs",
-  "main": "index.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
   "publishConfig": {
     "access": "public"
   },
+  "scripts": {
+    "build": "rollup -c rollup.config.mjs"
+  },
   "author": "Inrupt <devtools-team@inrupt.com>",
   "license": "MIT",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js"
+    }
+  },
   "dependencies": {
     "@inrupt/solid-client": "^1.23.3",
+    "@inrupt/solid-client-authn-browser": "^1.12.2",
     "@inrupt/solid-client-authn-node": "^1.12.2",
-    "dotenv": "^16.0.3"
```

Comment:
@chelseapinka dotenv should be a dependency of this package, as we directly use it.